<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DNPy</class>
 <widget class="QDialog" name="DNPy">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>DNPy - ODNP evaluation</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <property name="spacing">
      <number>6</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item alignment="Qt::AlignTop">
        <widget class="QGroupBox" name="generalGroup">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>General</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <property name="rowWrapPolicy">
           <enum>QFormLayout::DontWrapRows</enum>
          </property>
          <property name="verticalSpacing">
           <number>4</number>
          </property>
          <property name="topMargin">
           <number>9</number>
          </property>
          <property name="bottomMargin">
           <number>2</number>
          </property>
          <item row="1" column="0">
           <widget class="QLabel" name="pathLabel">
            <property name="text">
             <string>Experiment folder</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QToolButton" name="pathButton">
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="path"/>
            </item>
           </layout>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="powerFileLabel">
            <property name="text">
             <string>Powers file (optional)</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QToolButton" name="toolButton">
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="powerFile"/>
            </item>
           </layout>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="t1CalcLabel_3">
            <property name="text">
             <string>Auto phasing routine</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QComboBox" name="apCombo">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="frame">
             <bool>true</bool>
            </property>
            <item>
             <property name="text">
              <string>First experiment</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>All experiments</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>No auto phasing</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="ftWindowLabel">
            <property name="text">
             <string>Fourier transformation window [Hz]</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QSpinBox" name="ftWindow">
            <property name="maximum">
             <number>10000</number>
            </property>
            <property name="singleStep">
             <number>10</number>
            </property>
            <property name="value">
             <number>200</number>
            </property>
           </widget>
          </item>
          <item row="8" column="1">
           <widget class="QSpinBox" name="lB">
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="singleStep">
             <number>100</number>
            </property>
            <property name="value">
             <number>2</number>
            </property>
           </widget>
          </item>
          <item row="10" column="0">
           <widget class="QLabel" name="evalPathLabel">
            <property name="text">
             <string>Evaluation folder name</string>
            </property>
           </widget>
          </item>
          <item row="10" column="1">
           <widget class="QLineEdit" name="evalPath">
            <property name="text">
             <string>eval</string>
            </property>
           </widget>
          </item>
          <item row="12" column="0">
           <widget class="QLabel" name="mistLabel">
            <property name="text">
             <string>Misc</string>
            </property>
           </widget>
          </item>
          <item row="12" column="1">
           <widget class="QWidget" name="miscWidget" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>100</height>
             </size>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QCheckBox" name="basCor">
               <property name="text">
                <string>Baseline correction</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
               <property name="tristate">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="offCor">
               <property name="text">
                <string>Offset correction</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="dumpToCsv">
               <property name="text">
                <string>Export results to csv</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="kSigmaCalc">
               <property name="text">
                <string>Calculate kSigma</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="Debug">
               <property name="text">
                <string>Debug</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QSpinBox" name="maxWin">
            <property name="maximum">
             <number>100000</number>
            </property>
            <property name="singleStep">
             <number>100</number>
            </property>
            <property name="value">
             <number>1000</number>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="maxWinLabel">
            <property name="text">
             <string>Maximum frequency tolerance [Hz]</string>
            </property>
           </widget>
          </item>
          <item row="8" column="0">
           <widget class="QLabel" name="lBLabel">
            <property name="text">
             <string>Line broadening [Hz]</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item alignment="Qt::AlignTop">
        <widget class="QGroupBox" name="dnpGroup">
         <property name="title">
          <string>DNP</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
         <layout class="QFormLayout" name="formLayout_2">
          <property name="verticalSpacing">
           <number>4</number>
          </property>
          <property name="bottomMargin">
           <number>2</number>
          </property>
          <item row="1" column="0">
           <widget class="QLabel" name="lSLabel">
            <property name="text">
             <string>DNP left shift number</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="lS">
            <property name="maximum">
             <number>1000</number>
            </property>
            <property name="value">
             <number>70</number>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="rSLabel">
            <property name="text">
             <string>DNP right shift number</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QSpinBox" name="rS">
            <property name="maximum">
             <number>1000</number>
            </property>
            <property name="value">
             <number>110</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item alignment="Qt::AlignTop">
        <widget class="QGroupBox" name="t1SeriesEval">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="sizeIncrement">
          <size>
           <width>1</width>
           <height>2</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>1</width>
           <height>1</height>
          </size>
         </property>
         <property name="title">
          <string>T1</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <layout class="QFormLayout" name="formLayout_3">
          <property name="verticalSpacing">
           <number>4</number>
          </property>
          <property name="bottomMargin">
           <number>2</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="t1CalcLabel">
            <property name="text">
             <string>T1 calculation data</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="lSt1Label">
            <property name="text">
             <string>T1 left shift number</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="rSt1Label">
            <property name="text">
             <string>T1 right shift number</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="t1CalcCombo">
            <item>
             <property name="text">
              <string>Phase cycled real</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Phase cycled magnitude</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Real</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Magnitude</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QSpinBox" name="lSt1">
            <property name="maximum">
             <number>1000</number>
            </property>
            <property name="value">
             <number>70</number>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QSpinBox" name="rSt1">
            <property name="maximum">
             <number>1000</number>
            </property>
            <property name="value">
             <number>75</number>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QSpinBox" name="t1SeriesPolDeg">
            <property name="maximum">
             <number>10</number>
            </property>
            <property name="value">
             <number>1</number>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="t1SeriesPolDegLabel">
            <property name="text">
             <string>T1 series polynomial degree</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="makeFigs">
         <property name="title">
          <string>Figures</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <layout class="QFormLayout" name="formLayout_4">
          <property name="verticalSpacing">
           <number>4</number>
          </property>
          <property name="bottomMargin">
           <number>2</number>
          </property>
          <item row="2" column="0">
           <widget class="QLabel" name="figSizeLabel">
            <property name="text">
             <string>Figure size (width, height) [cm]</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QWidget" name="widget_2" native="true">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>27</height>
             </size>
            </property>
            <widget class="QSpinBox" name="figWidth">
             <property name="geometry">
              <rect>
               <x>50</x>
               <y>0</y>
               <width>51</width>
               <height>24</height>
              </rect>
             </property>
             <property name="value">
              <number>13</number>
             </property>
            </widget>
            <widget class="QSpinBox" name="figHeight">
             <property name="geometry">
              <rect>
               <x>130</x>
               <y>0</y>
               <width>51</width>
               <height>24</height>
              </rect>
             </property>
             <property name="value">
              <number>8</number>
             </property>
            </widget>
            <widget class="QLabel" name="label_2">
             <property name="geometry">
              <rect>
               <x>110</x>
               <y>0</y>
               <width>16</width>
               <height>21</height>
              </rect>
             </property>
             <property name="text">
              <string>x</string>
             </property>
            </widget>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>DPI [ppi]</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QSpinBox" name="figDpi">
            <property name="maximum">
             <number>1000</number>
            </property>
            <property name="singleStep">
             <number>10</number>
            </property>
            <property name="value">
             <number>250</number>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QWidget" name="formatsWidget" native="true">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QCheckBox" name="jpg">
               <property name="text">
                <string>JPEG</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="png">
               <property name="text">
                <string>PNG</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="pdf">
               <property name="text">
                <string>PDF</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="eps">
               <property name="text">
                <string>EPS</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="plotExtsLabel">
            <property name="text">
             <string>Formats</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QPushButton" name="cancelButton">
           <property name="text">
            <string>Cancel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="startButton">
           <property name="text">
            <string>Start</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="resultsLabel">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Results</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextBrowser" name="resultsBrowser">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="lineWidth">
          <number>1</number>
         </property>
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="CopyrightBox" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;ui/logo_red_shade_nostroke_120_small.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;a href=&quot;http://www.spintoolbox.com/&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;SpinToolbox.com&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Code and design by M. Hadi Timachi</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>pathButton</tabstop>
  <tabstop>path</tabstop>
  <tabstop>toolButton</tabstop>
  <tabstop>powerFile</tabstop>
  <tabstop>apCombo</tabstop>
  <tabstop>ftWindow</tabstop>
  <tabstop>maxWin</tabstop>
  <tabstop>lB</tabstop>
  <tabstop>evalPath</tabstop>
  <tabstop>basCor</tabstop>
  <tabstop>offCor</tabstop>
  <tabstop>dumpToCsv</tabstop>
  <tabstop>kSigmaCalc</tabstop>
  <tabstop>Debug</tabstop>
  <tabstop>lS</tabstop>
  <tabstop>rS</tabstop>
  <tabstop>t1SeriesEval</tabstop>
  <tabstop>t1CalcCombo</tabstop>
  <tabstop>lSt1</tabstop>
  <tabstop>rSt1</tabstop>
  <tabstop>t1SeriesPolDeg</tabstop>
  <tabstop>makeFigs</tabstop>
  <tabstop>jpg</tabstop>
  <tabstop>png</tabstop>
  <tabstop>pdf</tabstop>
  <tabstop>eps</tabstop>
  <tabstop>figWidth</tabstop>
  <tabstop>figHeight</tabstop>
  <tabstop>figDpi</tabstop>
  <tabstop>startButton</tabstop>
  <tabstop>cancelButton</tabstop>
  <tabstop>resultsBrowser</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
